projects/Mallard/src/helpers/push-notifications.ts

```diff
@@ -17,6 +17,7 @@ const pushNotifcationRegistration = () =>
             if (key) {
                 const screenSize = imageForScreenSize()
                 downloadAndUnzipIssue(key, screenSize)
+                clearOldIssues()
```

Comment:
I know this function will only delete old issues right now, but should we call it prior to calling `downloadAndUnzipIssue` in the event there is any regressions in that function in the future so that the newly downloaded issue isn't wiped?

Reply:
Just to be clear, you're suggesting to just move `await` it and move it to the line above?

Reply:
yup.

Reply:
I think I would prefer to move towards more comprehensive unit tests to ensure this doesnt happen.